gallery.html

```diff
@@ -0,0 +1,186 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title> Project Gallery </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&family=Mali:wght@600&family=Zilla+Slab:wght@700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+    <main class="gallery-container" id="gallery-container">    
+      <h1 class="gallery-title"> Project Gallery </h1>
+        <div class="articles-container">
+         <div class="article-wrapper">
+          <img src="main-img.jpg" class="gallery-img"/>
```

Comment:
These images aren't currently within our assets folder. Are these placeholders? It's fine if so, but let's also add an alt attribute. Even if empty for now.